mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,289 @@
+            name: 'tenantIds'
+            description: |
+              GCIP tenant ids that are linked to the IAP resource. tenantIds could be a string
+              beginning with a number character to indicate authenticating with GCIP tenant flow,
+              or in the format of _ to indicate authenticating with GCIP agent flow. If agent flow
+              is used, tenantIds should only contain one single element, while for tenant flow,
+              tenantIds can contain multiple elements.
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::String
+            name: 'loginPageUri'
+            description: |
+              Login page URI associated with the GCIP tenants. Typically, all resources within
+              the same project share the same login page, though it could be overridden at the
+              sub resource level.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'corsSettings'
+        description: |
+          Configuration to allow cross-origin requests via IAP.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'allowHttpOptions'
+            description: |
+              Configuration to allow HTTP OPTIONS calls to skip authorization.
+              If undefined, IAP will not apply any special logic to OPTIONS requests.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'oauthSettings'
+        description: |
+          Settings to configure IAP's OAuth behavior.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'loginHint'
+            description: |
+              Domain hint to send as hd=? parameter in OAuth request flow.
+              Enables redirect to primary IDP by skipping Google's login screen.
+              (https://developers.google.com/identity/protocols/OpenIDConnect#hd-param)
+              Note: IAP does not verify that the id token's hd claim matches this value
+              since access behavior is managed by IAM policies.
+          - !ruby/object:Api::Type::Array
+            name: 'programmaticClients'
+            description: |
+              List of client ids allowed to use IAP programmatically.
+            item_type: Api::Type::String
+      - !ruby/object:Api::Type::NestedObject
+        name: 'reauthSettings'
+        description: |
+          Settings to configure reauthentication policies in IAP.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'method'
+            required: true
+            description: |
+              Reauth method requested. The possible values are:
+
+              * `METHOD_UNSPECIFIED`: Reauthentication disabled.
+              * `LOGIN`: Prompts the user to log in again.
+              * `SECURE_KEY`: User must use their secure key 2nd factor device.
+              * `ENROLLED_SECOND_FACTORS`: User can use any enabled 2nd factor.
+            values:
+              - :METHOD_UNSPECIFIED
```

Comment:
`xxxx_UNSPECIFIED` is meaningless and can be removed.

Reply:
Removed `XXXX_UNSPECIFIED`